### PR TITLE
Ready: Upgrades and Remove Bower

### PR DESCRIPTION
**Pull request by donaldwasserman**
Hey There - 

This solves #17. I also updated various ember-cli dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,54 +2,59 @@
   "name": "ember-cli-opbeat",
   "version": "0.0.7",
   "description": "Add Opbeat exception logging to your Ember app.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "",
```
Comment (olleolleolle) [inline package.json]: This key is now duplicated. (Nitpicking a single thing to prove that I read some of this.)

**Comment by donaldwasserman** [comment]
Yeah - using Ember cli to update anything is such a pain. Also, I have to add an if statement to make sure there is a bower.json. 

I'll update it some more over the weeekend.

**Comment by broerse** [comment]
Perhaps also add something like this:

https://github.com/pouchdb-community/ember-pouch/blob/master/index.js#L15

because you can add but can't remove,

**Comment by donaldwasserman** [comment]
Ember-bootstrap handles it this way: 

https://github.com/kaliber5/ember-bootstrap/blob/71f3450e1490902e4e891a20f36ad139d2eb2bc4/blueprints/ember-bootstrap/index.js#L70 

I'll see if I can get it to work across a few of my projects, and provide some sort of fallback with the ui output if necessary.

**Comment by broerse** [comment]
Saw that but if you use other modules in your App that need the same bower modules you use removing is bad. In this case I think this is not likely so do the remove anyway perhaps?

**Comment by broerse** [comment]
Your code `if (bowerDeps['pouchdb`  => 'if (bowerDeps['opbeat-js' ;-)

**Comment by donaldwasserman** [comment]
@broerse @olleolleolle 

Thanks for all the help and apologies for the delays. I am currently running a client app in production from my fork. Seems like it's working awesomely.

Thanks for the willingness to take a look a look at this.

**Comment by nanderss** [comment]
@mriska @olleolleolle Should we merge this?

**Comment by donaldwasserman** [comment]
Thanks for merging this, although does this code need to be changed with the upcoming switch to elastic?

**Comment by olleolleolle** [comment]
That’d take more changes. We still need this change until May.

**Pull request merged**